runtest.sh

```diff
@@ -15,4 +15,10 @@ echo "\n||| Testing nix flake checks"
 nix --option sandbox false \
     build --override-input haskell-flake path:${FLAKE} -L .#check
 
+echo "\n||| Testing docs"
+nix --option sandbox false \
+    build --override-input haskell-flake path:${FLAKE} \
+    -L --show-trace \
+    github:hercules-ci/flake.parts-website#checks.x86_64-linux.linkcheck
```

Comment:
I'd remove `-L --show-trace` since they are not used in the commands above (or is there a reason for their inclusion, aside from debugging purposes?). Also, must sandbox be disabled for this build?

Reply:
The sandbox was disabled in an earlier command, so I assumed that you run this in a setup where that's necessary. This build works fine in the sandbox, but I've only ever built it in a nice NixOS-based sandbox.

The verbosity is helpful when this runs in CI. You'll only look at it when it fails, and then you want all the info. Module system errors tend to need `--show-trace`, at least until we have
 - https://github.com/NixOS/nix/issues/7553

Reply:
A proper CI solves this problem in the UI, but admittedly consuming an unpinned input like the site here isn't possible yet in Hercules.

Reply:
Anyway, I've changed it so that logs only appear when there's a problem. Still not great if the builder hangs, but that's unlikely to be a problem here.